hon.py

```diff
@@ -256,6 +256,8 @@ async def async_get_state(self, mac, typeName, returnAllData = False, loop=False
                 return json.loads(text)
             
             json_data = json.loads(text)["payload"]["shadow"]["parameters"]
```

Comment:
I don't know the impact of this on other integrations. Can you do it in another way?